src/ray/object_manager/ownership_based_object_directory.h

```diff
@@ -64,6 +64,8 @@ class OwnershipBasedObjectDirectory : public ObjectDirectory {
       const ObjectID &object_id, const NodeID &node_id,
       const object_manager::protocol::ObjectInfoT &object_info) override;
 
+  void RecordMetrics(uint64_t duration_ms) override;
```

Comment:
I opted to pass through the duration between metric recordings instead of having the object directory maintain the same "last recorded" logic as the raylet.